cosmos/job/drm/DRM_Base.py

```diff
@@ -67,7 +68,8 @@ def kill_tasks(self, tasks):
             self.kill(t)
 
     def cleanup_task(self, task):
-        pass
+        if task.drm_jobID and task.status not in completed_task_statuses:
+            self.kill(task)
```

Comment:
I hoisted this from `drm_k8s_jobs.py`, whose implementation seemed more broadly applicable